packages/create-skill-app/create-app.ts

```diff
@@ -127,6 +140,9 @@ export async function createApp({
         case 'README-template.md': {
           return 'README.md'
         }
+        case 'package-template.json': {
+          return 'package.json'
+        }
```

Comment:
we could source package.json from a template, but I haven't done that since having two of them seems cumbersome and easy to miss...